governance/polls/Adjust the System Surplus Buffer - April 5, 2021.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Increase the System Surplus Buffer - April 5, 2021
+summary: Signal your support or opposition for adjusting the Surplus Buffer 
+discussion_link: https://forum.makerdao.com/t/signal-request-adjust-the-surplus-buffer-march-april-2021/6979
+vote_type: Ranked Choice IRV
+categories:
+   - Risk Variable
+options:
+   0: Abstain
+   1: 30 million DAI (no change)
+   2: 60 million DAI (+30M)
+   3: 60 million DAI (+30M) while trying to `flap` 25% of the collected fees 
+---
+# Poll: Increase the System Surplus Buffer - April 5, 2021
+
+The Governance Facilitator(s) have placed a Governance Poll into the voting system on behalf of the Maker Governance Community. This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, April 5 at 16:00 UTC.
+
+**This is an instant-runoff vote. In an instant-runoff vote, you may rank any number of options in the order of preference. As a general rule, you should rank every option you would be willing to vote for if it was included in an executive vote. By failing to rank an option you signal that you are not willing to vote for it in an executive vote. Alternatively, if you would accept any of the listed options and have no preference as to which, you should vote 'Abstain' as your first choice and rank no other choices.**
+
+# Review
+
+MKR Token Holders may vote in this poll to keep or adjust the [System Surplus Buffer](https://community-development.makerdao.com/en/learn/governance/param-system-surplus-buffer) (also referred to as the 'System Surplus'):
+* 30 million DAI (no change)
+* 60 million DAI (+30 MM)
+* 60 million DAI (+30 MM) while trying to `flap` 25% of the collected fees 
+
+For the `flap` option we will set up an IAM with a target date aiming at directing just 75% of the new fees collected into the System Surplus by slowly increasing the `hump` so from time to time the the System Surplus is higher than `hump + bump` and a `flap` auction is triggered. **Governance cannot give any guarantee we will eventually burn 25% as there are a lot of factors which will influence of the actual System Surplus.**
```

Comment:
i am not too satisfied about this section - but probably good as a first attempt.